src/DependencyInjection/Compiler/ExtensionCompilerPass.php

```diff
@@ -31,16 +31,20 @@
  *     extends: array<class-string, string>,
  *     instanceof: array<class-string, string>,
  *     uses: array<class-string, string>,
+ *     admin_implements: array<class-string, string>,
+ *     admin_extends: array<class-string, string>,
+ *     admin_instanceof: array<class-string, string>,
+ *     admin_uses: array<class-string, string>,
  *     priority: int,
  * }>
  * @phpstan-type FlattenExtensionMap = array{
  *     global: array<string, array<string, array{priority: int}>>,
  *     excludes: array<string, array<string, array{priority: int}>>,
  *     admins: array<string, array<string, array{priority: int}>>,
- *     implements: array<string, array<class-string, array{priority: int}>>,
```

Comment:
its correct to remove those from the `FlattenExtensionMap`? They only appear in `ExtensionMap`?